src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -491,13 +493,13 @@ public void visitToken(DetailAST ast) {
      * Checks if current method's return type or variable's type is verifiable
      * according to <b>memberModifiers</b> option.
      *
-     * @param methodOrVariableDef METHOD_DEF or VARIABLE_DEF ast node.
+     * @param ast METHOD_DEF or VARIABLE_DEF ast node.
      * @return true if member is verifiable according to <b>memberModifiers</b> option.
      */
-    private boolean isVerifiable(DetailAST methodOrVariableDef) {
+    private boolean isVerifiable(DetailAST ast) {
         boolean result = true;
         if (!memberModifiers.isEmpty()) {
-            final DetailAST modifiersAst = methodOrVariableDef
+            final DetailAST modifiersAst = ast
```

Comment:
Now these changes are not related to issue we are solving. So, you must either:

1. Revert these changes
2. Make these as a separate commit, something like `minor: refactor...`

I suggest (1), to help ease review in this PR; the less changes, the easier the review process, and increased confidence on reviewer's part of changes in contributor's PR.

Reply:
Okay, I am doing it.